src/cljs/harja/views/urakka/aikataulu.cljs

```diff
@@ -148,6 +149,37 @@
     :epaonnistui-fn #(viesti/nayta! "Tallennus epäonnistui!"
                                     :warning
                                     viesti/viestin-nayttoaika-lyhyt)}))
+
+(defn valinnat [ur]
+  (let [{aikajana? :nayta-aikajana?
+         jarjestys :jarjestys
+         :as valinnat} @tiedot/valinnat]
+    [:span.aikataulu-valinnat
+     [valinnat/urakan-vuosi ur]
+     [valinnat/yllapitokohteen-kohdenumero yllapito-tiedot/kohdenumero]
+     [valinnat/tienumero yllapito-tiedot/tienumero]
+
+     [:span.label-ja-kentta-puolikas
```

Comment:
Voisiko tässä käyttää tee-otsikollinen-kentta komponenttia? Esim. urakan valinnoissa on käytetty näin:

[tee-otsikollinen-kentta
   {:otsikko "Tienumero"
    :kentta-params {:tyyppi :numero :placeholder "Rajaa tienumerolla" :kokonaisluku? true}
    :arvo-atom tienumero-atom
    :luokka "label-ja-kentta-puolikas"}
   "Tienumero"]